app/models/attend.rb

```diff
@@ -0,0 +1,8 @@
+class Attend < ApplicationRecord
```

Comment:
Please remove the model - we don't want to store the attendee information in our systems, we'll use nvite for that.